connectors/egeria/Makefile

```diff
@@ -10,6 +10,7 @@ build:
 	mvn -B clean package -DskipTests
 
 docker-build: build
+	docker build . -t ${IMG} -f Dockerfile
```

Comment:
Why do we need `-f Dockerfile`

Reply:
You're right we don't as `-f Dockerfile` is the default. Should we leave it so it's clear which file is taken?

Reply:
Having it makes it less clear to me, as if you do something special although you don't. Maybe it's just me.